### DISCUSSION TITLE
Scaler Discord Token Tracker

### DISCUSSION THREAD

**Discussion by taanii** [scaleracademy/open-source-projects#102] [Ideas]
A simple dashboard which can have the following features:

 - Scaler Discord store
 - Total tokens
 - Tokens spent
 - Token earned by X (X can be game, quiz, or challenge)
 a. Challenges
 b. Quizes
 c. Games
 d. Meme challenge
 e. Tech daily
 - Ongoing challenges
 - Upcoming challenges
 - Schedule of quizzes, games, activities etc
 
 Every discord community members needs this feature😅

**Comment by MrKrishnaAgarwal** [comment]
Yes!

**Comment by prathambagadiya** [comment]
Yes Every DIscord community members needed that feature

**Comment by jyotirmaya907** [comment]
Nice feature it will help a lot of members about tokens

**Comment by christusdsouza** [comment]
Neat idea, I guess scaler already has such a bot so additional features could be integrated!
